src/backend/distributed/metadata/node_metadata.c

```diff
@@ -483,11 +485,16 @@ citus_disable_node(PG_FUNCTION_ARGS)
 		ereport(ERROR, (errcode(ERRCODE_OBJECT_NOT_IN_PREREQUISITE_STATE),
 						errmsg("disabling the first worker node in the "
 							   "metadata is not allowed"),
-						errhint("You can force disabling node, but this operation "
-								"might cause replicated shards to diverge: SELECT "
-								"citus_disable_node('%s', %d, force:=true);",
-								workerNode->workerName,
-								nodePort)));
+						errhint("You can force disabling node, SELECT "
+								"citus_disable_node('%s', %d, force:=true, "
+								"synchronous:=true); "
+								"Passing synchronous:=false might cause replicated shards "
+								"to diverge.", workerNode->workerName, nodePort),
+						errdetail("Citus uses the first worker node in the "
+								  "metadata for certain internal operations when "
+								  "replicated tables are modified. Synchronous mode "
+								  "ensures the first worker node is accurately "
+								  "visible to all nodes.")));
```

Comment:
Maybe something like: Synchronous mode ensures that all nodes have the same view of the first worker node, which is used for certain locking operations